articles/enroll-hosts.md

```diff
@@ -138,15 +138,13 @@ In the Google Admin console:
 
 ### Unenroll
 
-You can unenroll a macOS, Windows, or Linux host from Fleet (iOS, iPadOS, and Android hosts coming soon).
-
 1. Determine if your host has MDM features turned on by looking at the **MDM status** on the host's **Host details** page. 
 
-2. For macOS hosts with MDM turned on, select **Actions > Turn off MDM** to turn MDM off. For Windows hosts with MDM turned on, follow the [instructions for turning off MDM](https://fleetdm.com/guides/windows-mdm-setup#turn-off-windows-mdm). For iOS, iPadOS hosts with MDM turned on, select **Actions > Turn off MDM**.
+2. For macOS hosts with MDM turned on, select **Actions > Turn off MDM** to turn MDM off. For Windows hosts with MDM turned on, download and run the [turn off MDM script](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/windows/scripts/turn-off-mdm.ps1) on the host. For iOS/iPadOS and Android hosts, select **Actions > Unenroll**.
```

Comment:
```suggestion
2. For macOS, Windows, iOS/iPadOS, and Android hosts:
  - For macOS with MDM turned on, select **Actions > Turn off MDM** to turn MDM off. 
  - For Windows hosts with MDM turned on, download and run the [turn off MDM script](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/windows/scripts/turn-off-mdm.ps1) on the host. 
  - For iOS/iPadOS and Android hosts, select **Actions > Unenroll**.
```

Reply:
@noahtalerman, what do you think of this to make it easier to scan for instructions for specific operating systems?

Reply:
I like it! I tweaked your suggestion.